cmd/podman/wait.go

```diff
@@ -41,7 +42,15 @@ func waitCmd(c *cli.Context) error {
 	}
 
 	var lastError error
-	for _, container := range c.Args() {
+	if c.Bool("latest") {
+		latestCtr, err := runtime.GetLatestContainer()
+		if err != nil {
+			return errors.Wrapf(err, "unable to wait on latest container")
```

Comment:
nit.  Is this valid?  I thought the only errors we received back were containers not found types of errors?  I didn't see a time out condition.

Reply:
at this juncture, the user has asked to wait on the latest container.  We're telling them, we couldn't wait on the latest container and it should add in the actual error (like no such container)

Reply:
Ugh, Brain Fart.  I was thinking there was no wait in the function rather than the user had asked to wait.  Ignore me, at least on this one!  ;^)